hack/istio/multicluster/deploy-kiali.sh

```diff
@@ -68,29 +74,49 @@ deploy_kiali() {
   if [ "${KIALI_USE_DEV_IMAGE}" == "true" ]; then
     if [ "${MANAGE_KIND}" == "true" ]; then
       echo "Pushing the images into the cluster..."
-      make -e DORP="${DORP}" -e CLUSTER_TYPE="kind" -e KIND_NAME="${cluster_name}" CLUSTER_REPO=localhost cluster-push-kiali
-      helm_args+=('--set deployment.image_pull_policy="Never"')
-      helm_args+=("--set deployment.image_name=localhost/kiali/kiali --set deployment.image_version=dev")
+      make -e -C "${KIALI_REPO_ROOT}" DORP="${DORP}" CLUSTER_TYPE="kind" KIND_NAME="${cluster_name}" cluster-push-kiali
```

Comment:
looks like the -e is in the wrong spot .  "-e" is for env var setting.. which is the DORP var.. so you want this (note that some make impls the -e is optional.. either get rid of all the `-e` or put them for everything. I honestly don't know why make has this -e because I've always seen it optional.

```suggestion
      make -C "${KIALI_REPO_ROOT}" -e DORP="${DORP}" -e CLUSTER_TYPE="kind" -e KIND_NAME="${cluster_name}" cluster-push-kiali
```

Reply:
Come to think of it, it could be if you do not specify `-e` maybe bash interprets that as env vars that it passes to the process. So it could be `make DORP=docker` is the same as `DORP=docker make ...` (that's just a guess).

Anyway, I would use `-e` to make it clear what it is going on, rather than omit it.

Reply:
I don't think you need to specify `-e` multiple times.
>        -e, --environment-overrides
>           Give variables taken from the environment precedence over variables from makefiles.

It's just a flag for the `make` command and not a variable argument?

Reply:
Ah yes... that explains things... https://www.gnu.org/software/make/manual/html_node/Environment.html